src/test/java/seedu/realodex/logic/commands/EditPersonDescriptorTest.java

```diff
@@ -55,6 +56,10 @@ public void equals() {
         // different tags -> returns false
         editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withTags(VALID_TAG_AMY).build();
         assertFalse(DESC_AMY.equals(editedAmy));
+
+        //different remarks -> returns false
+        editedAmy = new EditPersonDescriptorBuilder(DESC_AMY).withRemark(VALID_REMARK_BOB).build();
+        assertFalse(DESC_AMY.equals(editedAmy));
```

Comment:
Good check

Reply:
Not sure, maybe too long remark or something, I worried they may input something too long and UI may overflow or smth